src/client.h

```diff
@@ -121,6 +121,9 @@ class CClient : public QObject
 
     void Start();
     void Stop();
+    bool Connect ( QString strServerAddress, QString strServerName );
+    bool Disconnect();
+
     bool IsRunning() { return Sound.IsRunning(); }
     bool IsCallbackEntered() const { return Sound.IsCallbackEntered(); }
```

Comment:
IMO this is flawed. Sound should emit a signal if it started correctly, else emit an error signal. However, implementing this opens another pandora box.